models/user.js

```diff
@@ -32,6 +32,7 @@ module.exports = function User(ldapUserArray) {
                 userEntry.mail.map(function(mail) {
                     userModel.emails.push(mail);
                 });
+                userModel.emails = [...new Set(userModel.emails)];
```

Comment:
le spread operator (aka `...`)  est supporte a partir de la version 5.x... Du coup la doc annonce version 4... (juste au cas ou une vielle image docker n'a pas ete mise a jour 😄 )

Reply:
La doc dit: "⚠ this library "try" to use ES2015 (or ES6) capabilities, don't use it with nodejs under 4.x?"

Reply:
Yep, la doc devrait dire `under 5.x` 😄